onnxruntime/core/optimizer/graph_transformer_utils.cc

```diff
@@ -164,6 +162,16 @@ std::vector<std::unique_ptr<GraphTransformer>> GenerateTransformers(TransformerL
     }
     return transformers;
   }
+
+  // Some transformers have side-effect like result is not exactly same.
+  // These transformers could only be enabled by custom transformer list.
+#ifndef DISABLE_CONTRIB_OPS
+  if (level == TransformerLevel::Level2) {
+      std::unordered_set<std::string> cuda_execution_providers = {onnxruntime::kCudaExecutionProvider};
+      transformers.emplace_back(onnxruntime::make_unique<GeluApproximation>(cuda_execution_providers));
+  }
+#endif
```

Comment:
I don't quite understand why GeluApproximation is added to here.

Reply:
By default, there is no custom list, transformers will be returned in line 163. That does not include GeluApproximation.

Add GeluApproximation here so that it is included in a full set of transformers to filter by custom list below.